_data/conferences.yml

```
@@ -36,6 +36,9 @@
   end: 2020-06-26
   location: 🇪🇸 Barcelona, Spain
   cocoa-only: false
+  cfp:
+    link: https://addconf.typeform.com/to/UYLfh
```

There's a missing `u` at the end, the correct link is https://addconf.typeform.com/to/UYLfhu
```suggestion
    link: https://addconf.typeform.com/to/UYLfhu
```